src/pages/Tour.tsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from "react";
+import parseCSV from "../utils/parseCSV";
+import dateInPast from "../utils/checkDate";
+import TourDatesContainer from "../components/TourDates/TourDatesContainer";
+
+// Define a type for your data structure
+export type TourDateType = {
+  venue: string;
+  location: string;
+  event_date: string;
+  ticket_link: string;
+};
```

Comment:
As this is being used in multiple files perhaps we should have it in utils type file?